commons-numbers-primes/src/main/java/org/apache/commons/numbers/primes/Primes.java

```diff
@@ -81,12 +96,7 @@ public static int nextPrime(int n) {
 
         // prepare entry in the +2, +4 loop:
         // n should not be a multiple of 3
-        final int rem = n % 3;
-        if (0 == rem) { // if n % 3 == 0
-            n += 2; // n % 3 == 2
-        } else if (1 == rem) { // if n % 3 == 1
-            n += 4; // n % 3 == 2
-        }
+        n = nonMultipleOf3(n);
```

Comment:
I think this is refactoring for the sake of refactoring. The code comments explain what is occurring. The comments here also match the comments for the `+2, +4` adjustment that happens a few lines below within the loop.